base/cvd/cuttlefish/host/libs/web/android_build_api.cpp

```diff
@@ -232,8 +236,13 @@ Result<std::string> BuildApi::BuildStatus(const DeviceBuild& build) {
   std::string url = api_base_url_ + "/builds/" +
                     http_client->UrlEscape(build.id) + "/" +
                     http_client->UrlEscape(build.target);
+  bool firstQueryAdded = false;
   if (!api_key_.empty()) {
-    url += "?key=" + http_client->UrlEscape(api_key_);
+    url += "key=" + http_client->UrlEscape(api_key_);
```

Comment:
Why did you remove the leading `?`?

Reply:
Removed it by accident, thanks for the reminder!